torch/nn/modules/activation.py

```diff
@@ -30,9 +30,9 @@ class Threshold(Module):
         inplace: can optionally do the operation in-place. Default: ``False``
 
     Shape:
-        - Input: :math:`(N, *)` where `*` means, any number of additional
-          dimensions
-        - Output: :math:`(N, *)`, same shape as the input
+        - Input: :math:`(*)` where `*` means, any number of additional
```

Comment:
grammar nit: whoever wrote these initially put the comma  in the wrong place. Should be:
```
Input: :math:`(*)`, where `*` means any number of dimensions
```

other nit: no need for "additional" before "dimensions"

Same throughout all activation functions